src/types/wasm.ts

```
@@ -0,0 +1,367 @@
+    makeColor(_0: ArrayBuffer | Uint8Array | Uint8ClampedArray | Int8Array | string): any;
+    makePixels(_0: number): any;
+    makeBoolean(_0: boolean): any;
+    makeNull(): any;
+    makeUndefined(): any;
+  };
+  Synchronization: { make(_0: number, _1: ArrayBuffer | Uint8Array | Uint8ClampedArray | Int8Array | string, _2: number, _3: number, _4: number, _5: number, _6: number): any; makeFromUint8Array(_0: any): any };
+  Uint8Vector: { new (): Uint8Vector };
+  IConnector_WASM: { implement(_0: any): ImplementedIConnector_WASM; extend(_0: ArrayBuffer | Uint8Array | Uint8ClampedArray | Int8Array | string, _1: any): any };
+  ImplementedIConnector_WASM: {};
+  Spectoda_WASM: { implement(_0: any): ImplementedSpectoda_WASM; extend(_0: ArrayBuffer | Uint8Array | Uint8ClampedArray | Int8Array | string, _1: any): any };
+  ImplementedSpectoda_WASM: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)